.github/ISSUE_TEMPLATE.md

```diff
@@ -6,14 +6,17 @@ A summary of the issue.
 
```

Comment:
Before "Steps to reproduce" in the issue summary we should further direct the type of answer we are after – not any summary, we want to understand what is occuring, and what is expected.